Source/Relativity.Test.Helpers/Relativity.Test.Helpers/Mail/MailTrapMailHelper.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+
+namespace Relativity.Test.Helpers.Mail
+{
+	public class MailTrapMailHelper : IMailHelper
+	{
+		public Dictionary<string, string> RequestHeaders { get; set; }
+		public string BaseApiUrl => "https://mailtrap.io/";
+
+		public MailTrapMailHelper(string apiKey)
+		{
+			RequestHeaders = new Dictionary<string, string> { { "Api-Token", apiKey } };
+		}
+
+		/// <summary>
+		/// Gets all the inboxes on your account
+		/// </summary>
+		/// <returns></returns>
+		public List<IMailInboxModel> GetInboxes()
+		{
+			List<IMailInboxModel> inboxes = new List<IMailInboxModel>();
+
+			using (HttpClient client = new HttpClient())
+			{
+				client.BaseAddress = new Uri(BaseApiUrl);
+
+				foreach (var header in RequestHeaders)
+				{
+					client.DefaultRequestHeaders.Add(header.Key, header.Value);
+				}
+
+				string apiEndpoint = "api/v1/inboxes/";
+
+				HttpResponseMessage response = client.GetAsync(apiEndpoint).Result;
+				string json = response.Content.ReadAsStringAsync().Result;

```

Comment:
can we check if HTTP status is success before reading the content?

Reply:
Updated in latest commit